Watchdog/module.php

```diff
@@ -120,21 +120,16 @@ public function GetAlertTargets() {
 		
 		$watchTime = $this->GetWatchTime();
 		$watchTimeBorder = time() - $watchTime;
-		
 		$alertTargets = [];
 		
 		foreach ($targets as $target){
 									
 			$v = IPS_GetVariable($target["VariableID"]);
-			
-			if($this->ReadPropertyBoolean("CheckChange")) {
-				if($v['VariableChanged'] < $watchTimeBorder){
-					$alertTargets[] = array('Name' => $target["Name"], 'VariableID' => $target["VariableID"], 'LastUpdate' => $v['VariableChanged']);
-				}
-			} else {
-				if($v['VariableUpdated'] < $watchTimeBorder){
-					$alertTargets[] = array('Name' => $target["Name"], 'VariableID' => $target["VariableID"], 'LastUpdate' => $v['VariableUpdated']);
-				}
+
```

Comment:
So meinte ich den Umbau nicht. Lese am besten in einem vorbereitenden Schritt die relevante Zeit aus (Updated oder Changed, je nachdem) und erstelle nur einen Block mit der entsprechenden Prüfung.

Dein Umbau sollte übrigens funktionieren. Dies liegt allerdings nur daran, dass die Änderungszeit immer kleiner oder größer als die Updated-Zeit ist. Wenn man auf solche Hintergrundbedingungen angewiesen ist, dann sollte man einen entsprechenden Kommentar schreiben.